Jenkinsfile

```diff
@@ -260,7 +260,7 @@ node (node_label) {
 		// stage to run http scale test
 		if (http == "TRUE") {
 			env.WORKLOAD_PROPERTIES_FILE=HTTP_TEST_PROPERTY_FILE
-			env.WORKLOAD="ATS-SCALE-CI-HTTP"
+			env.WORKLOAD="INGRESS-PERFORMANCE"
```

Comment:
We might want to rename the properties file as well - https://github.com/cloud-bulldozer/scale-ci-pipeline/blob/master/properties-files/http.properties.